source/Octopus.Tentacle.Tests.Integration/Support/TentacleConfigurationsAttribute.cs

```diff
@@ -59,7 +61,7 @@ static class TentacleConfigurationTestCases
                     TentacleVersions.Current,
                     TentacleVersions.v5_0_15_LastOfVersion5,
                     TentacleVersions.v6_3_417_LastWithScriptServiceV1Only,
-                    TentacleVersions.v7_0_1_ScriptServiceV2Added
+                    TentacleVersions.v8_0_73_LastWithoutScriptServiceV3Alpha
```

Comment:
@nathanwoctopusdeploy , do you think it's worth keeping `v7_0_1_ScriptServiceV2Added` as well as this new one?

Reply:
Ah, sorry. Re-reading my question it didn't capture what I meant...

What I was asking was do we think we should also add `v7_0_1_ScriptServiceV2Added` as well as `v8_0_73_LastWithoutScriptServiceV3Alpha` in this list? I.e. increase the number of test cases.

As it stands, we are replacing v7 with v8. Which could be fine, as v8 has both.

Reply:
ah yep, fair call. I'll let @nathanwoctopusdeploy answer that one :)

Reply:
Yeh lets keep v7 and v8 as there are enough significant changes between v6 -> v7 and the v7 -> v8 to warrant testing both.

I would opt to go for the latest version of v7 e.g. `7.0.189` as well in place of 7.0.1

Reply:
no probs!

Reply:
Readded testing of v7 in 160be6c6

Reply:
turns the version was `7.1.189`... Also, `8.0.73` won't be the latest before this PR is merged, so updated to `8.0.81` in 720742f7